CMakeLists.txt

```diff
@@ -154,9 +161,13 @@ if (APPLE)
 else (APPLE)
    if (MINGW)
       # -mno-ms-bitfields see #22048
-      set(CMAKE_CXX_FLAGS_DEBUG   "-std=c++11 -mno-ms-bitfields -g")
-      set(CMAKE_CXX_FLAGS_RELEASE "-std=c++11 -mno-ms-bitfields -O2 -DNDEBUG -DQT_NO_DEBUG")
-      set(CMAKE_EXE_LINKER_FLAGS "-Wl,--large-address-aware")
+      set(CMAKE_CXX_FLAGS_DEBUG          "-std=c++11 -mno-ms-bitfields -g")
+      set(CMAKE_C_FLAGS_DEBUG            "-mno-ms-bitfields -g")
```

Comment:
The C_FLAGS are really needed? only for breakpad? if yes, could you put them around a BREAKPAD condition?